### PR TITLE
add header in project

### PR DESCRIPTION
**Pull request by MashaLoban**

### CODE REVIEW

File: Triphouse-project/index.html [hunk 1]
```diff
@@ -10,6 +10,29 @@
     <title>What do we offer</title>
 </head>
 <body>
+    <header>
+        <div class="wrapper__header">
+            <!-- <img src="../images/castelmezzano.jpg" alt="Главный экран"/> -->
+            <div class="wrapper__header_navigation">
+                <img src="../images/logo_vector.svg" alt="Logo"/>
+                <ul class="wrapper__header_list">
```
Comment (ghost) [inline Triphouse-project/index.html]: 2021-04-16_00h25_35

File: Triphouse-project/index.html [hunk 2]
```diff
@@ -10,6 +10,29 @@
...
+                         <a class="navigation__link" href="#">Attractions</a>
+                     </li>
+                    <li class="wrapper__header_list_item"><img src="../images/Night.svg" alt="Night"/></li>
+                    <li class="wrapper__header_list_item"><img src="../images/AccountCircle.svg" alt="Account"/></li>
+                </ul>
+            </div>
+            <div class="wrapper__header_text"><p>Discover stays </p><p>to live, work or just relax</p></div>
+            <div class="wrapper__header_download_items">
+                <img src="../images/google-play-badge.svg" alt="Logo_GP"/>
+                <img src="../images/Download_on_the_App_Store_Badge_US-UK_RGB_blk_092917 1.svg" alt="Logo_APPLE"/>
```
Comment (ghost) [inline Triphouse-project/index.html]: You have to rename a file with a long name.

File: Triphouse-project/index.html (hunk 1 above)
Comment (MashaLoban) [reply Triphouse-project/index.html]: deleted

File: Triphouse-project/index.html (hunk 2 above)
Comment (MashaLoban) [reply Triphouse-project/index.html]: renamed

**Pull request merged**